### PR TITLE
Update faq.rst to mention geod

### PR DESCRIPTION
**Pull request by jidanni**
See https://github.com/OSGeo/PROJ/issues/2644

### CODE REVIEW

File: docs/source/faq.rst [hunk 1]
```diff
@@ -135,7 +135,9 @@ input ordered as latitude/longitude  (typically with the EPSG dataset), however,
 internally PROJ expects an longitude/latitude ordering for all projections. This
 is generally hidden for users but in a few cases it is exposed at the surface
 level of PROJ, most prominently in the :program:`proj` utility which expects
-longitude/latitude ordering of input date (unless :option:`proj -r` is used).
+longitude/latitude ordering of input data (unless :option:`proj -r` is used).
+(And `geod`, which on the contrary expects latitude/longetude, and lacks any
```
Comment (busstoptaktik) [inline docs/source/faq.rst]: \`geod\` should probably be :program:\`geod\`.

Also, putting an entire sentence in parenthesis looks strange, especially as it bumps oddly against the parenthesis at the end of the previous sentence. I suggest something like
```rst
... (unless :option:`proj -r` is used), while :program:`geod`,
on the contrary, expects latitude/longitude, and does not support the `-r` option.
```

Also note the correction longetude -> longitude

**Review by busstoptaktik — CHANGES_REQUESTED** [review]
Needs some linguistic TLC

**Comment by jidanni** [comment]
I'm very grateful for anybody to take this anywhere they want to.

Regarding minus r. Note that if such an option was added to geod, it should be a no-op. And then a different option added to accept lon lat.

And then after adding these options, the article might just complain about contrasting defaults, without mentioning  lack of options.

**Comment by jidanni** [comment]
In https://github.com/OSGeo/PROJ/issues/2644#issuecomment-1731100116 I mention a scheme, where adding options to all the programs offered, could wallpaper over the worry about different defaults for different programs (that the PROJect distributes... Internal contradictions)

In fact the FAQ article could mention this scheme as a cherry note.

Of course implementing it: I'll leave that up to you guys :-)

File: docs/source/faq.rst (hunk 1 above)
Comment (rouault) [inline docs/source/faq.rst]:
```suggestion
(And :program:`geod`, which on the contrary expects latitude/longitude, and lacks any
```

**Pull request merged**